internal/service/imagebuilder/distribution_configuration_test.go

```diff
@@ -704,6 +767,53 @@ resource "aws_imagebuilder_distribution_configuration" "test" {
 `, rName, userId)
 }
 
+func testAccDistributionConfigurationDistributionAMIDistributionConfigurationLaunchPermissionOrganizationArnsConfig(rName string) string {
+	return fmt.Sprintf(`
+data "aws_region" "current" {}
+data "aws_partition" "current" {}
+
+resource "aws_organizations_organization" "test" {}
+
+resource "aws_imagebuilder_distribution_configuration" "test" {
+  name = %[1]q
+  distribution {
+    ami_distribution_configuration {
+      launch_permission {
+        organization_arns = [aws_organizations_organization.test.arn]
+      }
+    }
+    region = data.aws_region.current.name
+  }
+}
+`, rName)
+}
+
+func testAccDistributionConfigurationDistributionAMIDistributionConfigurationLaunchPermissionOrganizationalUnitArnsConfig(rName string) string {
+	return fmt.Sprintf(`
+data "aws_region" "current" {}
+data "aws_partition" "current" {}
+
+resource "aws_organizations_organization" "test" {}
+
+resource "aws_organizations_organizational_unit" "test" {
+	name = %[1]q
+	parent_id = aws_organizations_organization.test.roots[0].id
+}
+
+resource "aws_imagebuilder_distribution_configuration" "test" {
+	name = %[1]q
+	distribution {
+	  ami_distribution_configuration {
+		launch_permission {
+		  organizational_unit_arns = [aws_organizations_organizational_unit.test.arn]
+		}
+	  }
+	  region = data.aws_region.current.name
+	}
+  }
+  `, rName)
```

Comment:
I think we'll have a bunch of `terrafmt` errors here.

Reply:
@kamilturek I fixed the `terrafmt` issues + added docs + added changelog.